code/src/components/Overview.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react'
+
+const hungerLevel = {
+  1: 'You are full, but here are some recipes for later:',
+  2: 'You are feeling snacky, grab a cookie and check out these recipes: ',
+  3: 'You are HUNGRY. Hurry, decide what you want to eat, and run to the store! Link here:'
+}
+
+const Overview = ({ nameInput, preference, sortOfPasta, otherIngredient, hungerInput }) => {
+  return (
+    <div className="survey-container">
+      <section className="overview-container">
+        <h2 tabIndex="0">Thanks for participating {nameInput}!</h2>
+        <div className="overview-summary">
+          <p tabIndex="0">You are in the mood for some {preference} food</p>
+          <p tabIndex="0">You prefere {sortOfPasta}, with some {otherIngredient}, nice!</p>
+          <p tabIndex="0">{hungerLevel[hungerInput]} <a href="https://www.yummly.com/recipes?q=pasta&taste-pref-appended=true" target="_blank" rel="noreferrer">Recipes!</a></p>
```

Comment:
I like this idea that you give a link to recipies. Would be fun to get some different recepies depending on on the other input andon how hungry you said you were.

Reply:
I knooow! I wanted to do it a lot differently, I wanted to have if/else or something like that in every question.... but it got to komplex, I really tried figuring it out tho.